aptos-move/move-examples/defi/aptoswap/init/sources/pool_manager.move

```diff
@@ -0,0 +1,43 @@
+module aptoswap::init {
```

Comment:
This can be done in a followup but we should make a separate example for the object-based deployment flow